server.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"api-booking-time/config"
+	"api-booking-time/infrastructure/datastore"
+)
+
+var (
+	db *datastore.DbRepository
+)
+
+func main() {
+	config.ReadConfig()
+	db = datastore.OpenDb()
+
+	r := mux.NewRouter()
+	api := r.PathPrefix("/api/v1").Subrouter()
+	api.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		fmt.Fprintln(w, "api booking-time")
+	})
```

Comment:
That second parameter in the `HandlerFunc()` corresponds to a function that will handle the request.

In this case, it's logic corresponding to the controller layer (speaking of clean architecture, and in your project is actually in the `routes.go` file).

You already have there the `getCentres` function there, why you didn't handle that the same way as the `/centres` path?